plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/decoders/TimestampDecoder.java

```diff
@@ -63,7 +65,10 @@ public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output)
         else {
             LocalDateTime timestamp;
             if (value instanceof String) {
-                timestamp = ISO_DATE_TIME.parse((String) value, LocalDateTime::from);
+                String valueString = (String) value;
+                timestamp = Optional.ofNullable(Longs.tryParse(valueString))
```

Comment:
I would use regular `if`. Using `Optional.ofNullable` looks like an overkill to me.

Reply:
AC.